specs/v2-sketch.yaml

```diff
@@ -393,11 +428,8 @@ components:
         created_at:
           type: string
           format: date-time
-        source_config:
-          type: string
-        compiled_config:
-          type: string
-        #trigger:
+        trigger:
+          $ref: "#/components/schemas/TriggerSummary"
         idempotency_key:
           type: string
         #actor:
```

Comment:
Still need to decide on the shape for an actor.

Reply:
Also, do we want to include organization data in this response?

Reply:
I would be in favour of keeping the actor as "thin" as possible here. Just a UUID for now, we can expand later.

Reply:
That sounds reasonable, we'll definitely want more info later.

Reply:
Going to punt on the actor for now.